src/views/Picklist-Detail.vue

```diff
@@ -2,76 +2,186 @@
+import { JsonCSV } from '@/mixins/jsonToCsv'
+import emitter from '@/event-bus'
 
 export default defineComponent({
-  name: 'Settings',
+  name: 'PicklistDetail',
   components: {
     IonBackButton, 
     IonButton, 
     IonButtons, 
-    IonContent, 
-    IonFab, 
-    IonFabButton, 
+    IonContent,
+    IonFooter, 
     IonHeader, 
-    IonIcon, 
-    IonItemDivider, 
-    IonList, 
-    IonLabel, 
+    IonIcon,
+    IonItemDivider,
+    IonItemGroup, 
+    IonLabel,
+    IonList,
     IonPage, 
     IonTitle, 
     IonToolbar,
     PicklistDetailItem
   },
+  computed: {
+    ...mapGetters({
+      picklists: 'picklist/getPicklists',
+      picklistItem: 'picklist/getCurrent',
+    })
+  },
+  data() {
+    return {
+      picklistGroup: []
+    }
+  },
+  props: ['id'],
+  mixins: [JsonCSV],
+  mounted () {
+    // Sort picklist products alphabetically
+    // Used localeCompare to compare productName
+    this.picklistItem.pickingItemList.sort((a, b) => a.productName.localeCompare(b.productName, { sensitivity: 'base' }));
+    // Arrange picklist products alphabetically
+    const data = this.picklistItem.pickingItemList.reduce((r, e) => {
+      // To display "0th" index value i.e. first letter of product name
+      const alphabet = e.productName[0];
+      if (!r[alphabet]) r[alphabet] = { alphabet, record: [e] }
+      else r[alphabet].record.push(e);
+      return r;
+    }, {});
+    this.picklistGroup = Object.values(data);
+    emitter.on("export-finished", this.completePicklists)
+  },
   methods: {
-      async presentAlertMultipleButtons() {
+    async completePicklist(blob) {
+      const picklistChecked = this.picklistItem.pickingItemList.some((picklist) =>
+        picklist.isChecked
+      )
+      if (picklistChecked) {
+        const formData = new FormData();
+        const fileName = "CompletePicklist_" + Date.now() +".csv";
+        formData.append("uploadedFile", blob, fileName);
+        formData.append("configId", "IMP_UPD_PKLST_ITM_ST");
+
+        return this.store.dispatch("picklist/completePicklists", {
+          headers: {
+            'Content-Type': 'multipart/form-data;'
+          },
+          data: formData
+        }).then(() => {
+          this.router.push("/tabs/picklists");
+        })
+      } else {
+        showToast(translate("Something went wrong"));
+      }
+    },
+    async completeProductPicklist() {
       const alert = await alertController
         .create({
           header: this.$t("Complete picklist"),
-          buttons: [this.$t("Cancel"), this.$t("Complete")],
+          buttons: [
+            {
+              text: this.$t('Cancel'),
+              role: 'cancel'
+            },
+            {
+              text:this.$t('Complete'),
+              handler: () => {
+                if (this.picklistItem.pickingItemList.some((picklist) => picklist.isChecked)) this.generate();
+                else showToast(translate("No item has been picked"));
```

Comment:
I think this is not added to en.json. Please check all labels once

Reply:
Ok

Reply:
This label is internationalised in another MR
https://github.com/hotwax/pickingapp/pull/16/files